app/views/feedback.jade

```diff
@@ -8,7 +8,8 @@ html
 
     link(rel='stylesheet' href='styles/main.css')
     script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
-    script(type="text/javascript" src="/scripts/satisfaction.js")
+    script(type="text/javascript" src="https://code.jquery.com/jquery-2.2.3.min.js")
+    script(type="text/javascript" src="/scripts/satisfaction.min.js")
```

Comment:
I'm probably missing some context here, but why did you need a separate bundle?
Why are you loading this script in the head instead of at the end of the body like other app scripts?